### ISSUE TITLE
Remove all Year Filters in Host Dashboard Page and Only Rely on Custom Date Filters

### ISSUE THREAD

**Issue by SudharakaP** [opencollective/opencollective#5350]
This is suggested at, https://opencollective.slack.com/archives/C02C146LFCP/p1647642659927419

Currently we have yearly filters within the `Total Money Managed` and `Host Fees` sections. We could potentially remove these yearly filters and have only the filters at the top of the page and make sure the Total Money Managed and Host Fee graphs has date ranges dependent upon the filter that the user chose (just like the `Contributions and Expenses` section in the page). 

If there's no objections to this I'll do this improvement as I feel this will give the users better control of the data they want to see. 

cc: @Betree

**Comment by Betree** [comment]
Agree, it still confuses me that we have multiple date filters in there

**Comment by Betree** [comment]
Related: https://github.com/opencollective/opencollective/issues/5464
